term API doc: added referenced ACTIF sample file (#89)
acrolinx/platform-api: term-api.md

> - In both cases, creation or update, the return code of the service is 201 (Created) and the Location header gets the

Reviewer: ```suggestion
- For both creation and update, the return code is 201 (created) and the "Location" header gets the
```

  URL of the inserted/updated entry.

> - If the input data does not contain entry to create or update, the return code is 204 (No Content).

Reviewer: ```suggestion
- If the input data doesn't contain an entry that needs to be created or updated, you'll see a 204 (no content).
```

> - If the input data contains more than one entry (which is allowed), all entries in the input data will be created or updated respectively, but

Reviewer: ```suggestion
- If the input data contains more than one entry (which is allowed), all entries in the input data will be created or updated respectively. However,
```

  URL of the inserted/updated entry.

> - If no new entry was created or updated but it’s no error (for example, the input data could be empty and not contain any

Reviewer: the "but it's no error" part of this is confusing to me

Author: I shortened it.

  insertable entry), the return code is 204 (No Content).
- If the input data contains more than one entry (which is allowed), all entries in the input data will be created or updated respectively, but

>   the Location header will only contain the URL of one of them.

Reviewer: ```suggestion
  the "Location" header will only contain the URL of one of them.
```

Author: Changed.

> - If the input data contains more than one entry (which is allowed), all will be created or updated respectively, but

Reviewer: What is "all" referring to here? All entries?

Author: all = all entries in the input data. I clarified this in another commit.
